setup.py

```diff
@@ -37,7 +37,8 @@ def is_requirement(line):
         line.startswith('-r') or
         line.startswith('#') or
         line.startswith('-e') or
-        line.startswith('git+')
+        line.startswith('git+') or
+        line.startswith('-c')
```

Comment:
FYI:
```
return not (line == '' or line.startswith(('-r', '#', '-e', 'git+', '-c')))
```

Reply:
Huh, guess I missed the memo on that being added in Python 2.5.